### ISSUE TITLE
Inquiry about reproducing success rates in VTDexManip

### ISSUE THREAD

**Issue by hdjlcbz** [LQTS/VTDexManip#5]
Dear authors,
I have been trying to reproduce the success rates mentioned in your ICLR 2025 paper "VTDexManip: A DATASET AND BENCHMARK FOR VISUAL-TACTILE PRETRAINING AND DEXTEROUS MANIPULATION WITH REINFORCEMENT LEARNING" by running the train_agent.py to train the RL policy and then using eval_agent to measure the success rate. However, I found that the success rates I obtained are quite different from those reported in the paper.
I'm wondering if there are some specific configurations or settings that I might have missed, which could lead to such discrepancies. Could you please provide some guidance on the key configurations or additional steps that should be taken to achieve success rates similar to those in the paper?
Thank you very much for your time and help. I look forward to your reply.

**Comment by LQTS** [comment]
> Dear authors, I have been trying to reproduce the success rates mentioned in your ICLR 2025 paper "VTDexManip: A DATASET AND BENCHMARK FOR VISUAL-TACTILE PRETRAINING AND DEXTEROUS MANIPULATION WITH REINFORCEMENT LEARNING" by running the train_agent.py to train the RL policy and then using eval_agent to measure the success rate. However, I found that the success rates I obtained are quite different from those reported in the paper. I'm wondering if there are some specific configurations or settings that I might have missed, which could lead to such discrepancies. Could you please provide some guidance on the key configurations or additional steps that should be taken to achieve success rates similar to those in the paper? Thank you very much for your time and help. I look forward to your reply.

Thank you for your feedback! This issue has also been reported by other users, and we are currently investigating it. We will update you as soon as we have any progress. In the meantime, could you please share the specific data or reproduction results you obtained? That would help us diagnose and resolve the problem more directly.

**Comment by hdjlcbz** [comment]
I have trained the models using the provided YAML configurations and evaluated them without any changes to the configurations. For the evaluation, I have used the following command for the "bottle_cap-vt_all_cls" task under seen conditions:
python eval_agent.py --task bottle_cap-vt_all_cls --rl_device cuda:0 --resume_model runs/BottleCap/bottle_cap/bottle_cap-vt_all_cls/seed111/checkpoint/model_2000.pt --test --seed 111
I have also evaluated "Table Reorientation" and "In-hand Reorientation" using 3000 iterations, while the other tasks were evaluated using 2000 iterations. The success rates I obtained are as follows:
BottleCap Turning (Seen): 85.01%
Bimanual Hand-over (Seen): 22.75%
Table Reorientation (Seen): 63.84%
In-hand Reorientation (Seen): 60.54%
Faucet Screwing (Seen): 73.05%
Lever Sliding (Seen): 98.40%
I am planning to evaluate the unseen conditions next, following the configurations in your paper.
I would like to ask if my evaluation process is correct and if the success rates I obtained are within the expected range. If there are any additional steps or configurations that I should consider, I would greatly appreciate your guidance.
Thank you very much for your time and for sharing your excellent work.
